en-US/snippets/2017/sep2017_a.lang

```diff
@@ -0,0 +1,122 @@
+;Send Firefox to your mobile device for better browsing on the go.
+Send Firefox to your mobile device for better browsing on the go.
+
+
+# Snippet for Firefox Focus
+# This message lets users know they can trust Focus to keep their secret.
+;Browse for something secret
+Browse for something secret
+
+
+# Snippet for Firefox Focus
+# This message highlights the app's lightweight in the world of high cost of data plan, in addition to it being a private browser.
+;Try Firefox Focus, the lightweight mobile browser that lets you explore the web without a trace.
+Try Firefox Focus, the lightweight mobile browser that lets you explore the web without a trace.
+
+
+# Snippet for Firefox Focus
+# This message highlights that the mobile app is fast.
+;Want to browse faster on the go?
+Want to browse faster on the go?
+
+
+# Snippet for profiling age test
+# Link: https://addons.mozilla.org/firefox/
+# Power up" means to make better, to improve. The customization with add-ons will make the browser better.
+;Power up <a>Firefox with Add-ons.</a> Choose from dozens of customizations for privacy, social, gaming, and a whole lot more.
+Power up <a>Firefox with Add-ons.</a> Choose from dozens of customizations for privacy, social, gaming, and a whole lot more.
+
+
+# Snippet for profiling age test
+# Linke: https://www.mozilla.org/internet-health/
+# This message targets new users. "Webby journey" means all the browsing on the internet.
+# "Not all heroes wear capes" suggests heroes don't always look like those featured in movies and comic books. Ordinary people can be impactful by using a product made by Mozilla.
+;Hey, nice work choosing Firefox for your webby journeys. Not all heroes wear capes and by browsing with us, you're helping <a>build a better internet.</a>
+Hey, nice work choosing Firefox for your webby journeys. Not all heroes wear capes and by browsing with us, you're helping <a>build a better internet.</a>
+
+
+# Snippet for desktop
+# Link: https://support.mozilla.org/kb/private-browsing-use-firefox-without-history
+# This message lets users know how to enable Private browsing in Firefox so they can browse freely without fear of being tracked by third parties.
+;<a>Open a private window</a> in Firefox to browse freely and keep away the trackers.
+<a>Open a private window</a> in Firefox to browse freely and keep away the trackers.
+
+
+# Snippet for Firefox Accounts
+# Link: https://www.mozilla.org/firefox/accounts/
+# This message is for a user who doesn't have a Firefox account yet, and highlights the convenience of having an account will bring.
+;<a>Browse boldly.</a> Sign up for a Firefox Account to access your protected passwords, open tabs and bookmarks on all your devices.
+<a>Browse boldly.</a> Sign up for a Firefox Account to access your protected passwords, open tabs and bookmarks on all your devices.
+
+
+# Snippet for Firefox Accounts
```

Comment:
It took me a while to figure out the different between these two snippets. Maybe explain in the note that one is about creating a new account, the other signing in an existing account?